### ISSUE TITLE
GitHub Actions

### ISSUE THREAD

**Issue by bast** [coderefinery/testing#54]
I think we could simplify the lesson by demonstrating automated testing with GitHub actions:
- no new account to create
- nothing to enable and not a different tab to switch to
- better support for testing on macOS and Windows
- all you need to do is add a file, that's it
- we could point learners to the many examples for other languages

I can adjust lesson but don't want to change it now under @blindij feet.

But I think the minimum we need to do and this is what I would ask @blindij to do in the upcoming workshops is to at least mention it.

Otherwise we can look outdated. I still like and will use Travis but for many situations GH Actions is just simpler.

**Comment by annefou** [comment]
I agree. Travis seems an overkill in many simple projects. We should definitely teach it.

**Comment by annefou** [comment]
I mean github actions!

**Comment by rkdarst** [comment]
I really really like github actions, but really don't like the monopolization.  Still, seems like what we need to do.

I recently decided what I should do to avoid lock-in: I will try to avoid putting the test scripts in the github-specific YAML file, but in some makefile (or equivalent) with targets such as `test-install` and `test-run`, if it's not trivial.  So, at least porting to a new CI system is relatively easy.  But of course, then we lose some of the power of GHA.

Anyway, I would support mentioning the alternatives and lock-in potential, but fully switching to GHA. The easier it is to test, the more people who will, and I accept the trade off...
